skyportal/handlers/api/source.py

```diff
@@ -60,7 +62,13 @@ def get(self, source_id=None):
         simbad_class = self.get_query_argument('simbadClass', None)
         has_tns_name = self.get_query_argument('hasTNSname', None)
         total_matches = self.get_query_argument('totalMatches', None)
+        is_token_request = isinstance(self.current_user, Token)
         if source_id:
+            if is_token_request:
```

Comment:
So, we register both token and non-token views. Looks like we currently tally both in the view count, correct?

Reply:
Yes, we tally both token and browser views. The token views are tallied here in the source post handler, but for the front-end, we explicitly hit `/api/internal/source_views` with a POST whenever a non-cached source page is loaded